project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ package com.gu
 import sbt._
 
 object Dependencies {
-  val identityLibVersion = "3.96"
+  val identityLibVersion = "3.98"
```

Comment:
Where's the corresponding identity PR?

Reply:
https://github.com/guardian/identity/pull/745